polkadot/node/network/protocol/src/request_response/mod.rs

```diff
@@ -248,8 +248,8 @@ impl Protocol {
 				name,
 				fallback_names,
 				max_request_size: 1_000,
-				/// Responses are just confirmation, in essence not even a bit. So 100 seems
-				/// plenty.
+				// Responses are just confirmation, in essence not even a bit. So 100 seems
+				// plenty.
```

Comment:
```suggestion
				// Responses are just confirmation, in essence not even a bit. So 100 seems plenty.
```
This takes it just under the linewidth, but it seems there are other short linewraps in this file so feel free to ignore.